src/plugins/intel_cpu/CMakeLists.txt

```
@@ -34,7 +34,7 @@ elseif(AARCH64)
 endif()
 set(OV_CPU_ARM_TARGET_ARCH ${OV_CPU_ARM_TARGET_ARCH_DEFAULT})
 
-if(X86 OR X86_64 OR AARCH64)
+if(X86 OR X86_64 OR (AARCH64 AND NOT WIN32))
     # disable mlas with webassembly
     if(EMSCRIPTEN OR MINGW OR (CMAKE_COMPILER_IS_GNUCXX AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 7))
```

```suggestion
    if(EMSCRIPTEN OR WIN32 OR MINGW OR (CMAKE_COMPILER_IS_GNUCXX AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 7))
```
maybe it's better to move the check here?

---

Doesn't Windows on x86 support mlas?

---

corrected

Maybe we need to refactor these conditions to have 1 level of conditons..